ui_framework/components/modal/confirm_modal.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { KuiModal } from '../modal';
+import { KuiModalFooter } from './modal_footer';
+import { KuiModalHeader } from './modal_header';
+import { KuiModalBody } from './modal_body';
+import { KuiModalBodyText } from './modal_body_text';
+import { KuiButton } from '../index';
+
+export function KuiConfirmModal({ message, title, onCancel, onConfirm, cancelButtonText, confirmButtonText, ...rest }) {
```

Comment:
I prefer @cjcenizal's approach instead of receiving `props` then immediately destructuring (so keeping as-is, just splitting over multiple lines).